index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
Why is there a gap here?
```suggestion
console.log('line 13 ?');
```